storage/src/adb/any/variable.rs

```diff
@@ -0,0 +1,1227 @@
+//! An authenticated database (ADB) that provides succinct proofs of _any_ value ever associated
```

Comment:
nit:

```
//! An authenticated database (ADB) that provides succinct proofs of _any_ value ever associated
//! with a key, where values can have varying sizes.
//!
//! _If the stored values are always the same size, use [crate::adb::any::Any] instead._

Reply:
done

Reply:
It is showing as unchanged to me?

<img width="802" height="147" alt="Image" src="https://github.com/user-attachments/assets/4c7ce030-4fee-43d5-9bd9-f40c0dd53548" />